test-harness/gold-master-files/__path____user_username___method___GET__.json

```diff
@@ -10,7 +10,7 @@
     "statusText": "OK",
     "headers": {
       "content-type": [
-        "application/json; charset=utf-8"
```

Comment:
This reminds me, do we have any tests where the spec has `application/json; charset=utf-8` but they request has `accept: application/json`? I'd like to be very sure that works.

Reply:
Added in 2ebcaef

Reply:
5bc26a9